tools/sense_studio/sense_studio.py

```diff
@@ -562,9 +562,10 @@ def train_logreg():
 
         X = np.array(X)
         y = np.array(y)
-        logreg = LogisticRegression(C=0.1, class_weight=class_weight)
-        logreg.fit(X, y)
-        dump(logreg, logreg_path)
+        if len(class_weight) > 1:
```

Comment:
In `else`, I am thinking of creating a message or something which will be passed to template for the user.